COPYRIGHT

```diff
@@ -1,3 +1,6 @@
-Copyright 2008-2014, 2016, 2017, 2019, 2020 Association of Universities for Research in Astronomy, Inc. (AURA)
-Copyright 2015, 2017, 2018, 2019 The Trustees of Princeton University
-Copyright 2015, 2018, 2019 University of Washington
+Copyright 2015-2020, 2022-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+Copyright 2014-2023 The Trustees of Princeton University
+Copyright 2015-2016, 2018-2020, 2022-2023 University of Washington
+Copyright 2015, 2018-2020 The Board of Trustees of the Leland Stanford Junior University, through SLAC National Accelerator Laboratory
+Copyright 2015 The University of Tokyo
```

Comment:
As far as I can tell there is one commit from Tokyo and is 2 lines of code. Generally copyright assignment requires more of a contribution than that and that is why I did not add this line originally (I had already gone through the code to generate the original copyright file contents around 2020). Please remove.